### ISSUE TITLE
Crash with Fabric 0.13+

### ISSUE THREAD

**Issue by Xen0Xys** [Siphalor/mouse-wheelie#156]
**Version**
1.18.1
**Describe the bug**
The mod cause crash with fabric 0.13+

**To Reproduce**
Launch with fabric 0.13+
See crash

**Expected behavior**
Correct launch

**Crash report**
```
---- Minecraft Crash Report ----
// Uh... Did I do that?

Time: 2/22/22, 7:20 PM
Description: Initializing game

java.lang.RuntimeException: Mixin transformation of net.minecraft.class_312 failed
	at Not Enough Crashes deobfuscated stack trace.(1.18.1+build.12)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:252)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at net.minecraft.client.MinecraftClient.<init>(MinecraftClient:492)
	at net.minecraft.client.main.Main.main(Main:199)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:234)
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClassBytes(MixinTransformer.java:202)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:247)
	... 12 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionError: LVT in net/minecraft/class_312::method_1598(JDD)V has incompatible changes at opcode 312 in callback mixins.recording.replaymod.json:MixinMouseHelper->@Inject::handleReplayModScroll(JDDLorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;DF)V from mod (unknown).
 Expected: [D, F]
    Found: [D, D]
Available: [D, D, D, F]
	at org.spongepowered.asm.mixin.injection.callback.CallbackInjector.inject(CallbackInjector.java:547)
	at org.spongepowered.asm.mixin.injection.callback.CallbackInjector.inject(CallbackInjector.java:497)
	at org.spongepowered.asm.mixin.injection.code.Injector.inject(Injector.java:276)
	at org.spongepowered.asm.mixin.injection.struct.InjectionInfo.inject(InjectionInfo.java:445)
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.applyInjections(MixinTargetContext.java:1374)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyInjections(MixinApplicatorStandard.java:1052)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.applyMixin(MixinApplicatorStandard.java:400)
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:325)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:421)
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:403)
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363)
	... 15 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.getPostMixinClassByteArray(KnotClassDelegate.java:252)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.tryLoadClass(KnotClassDelegate.java:150)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:155)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
	at net.minecraft.class_310.<init>(class_310.java:492)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:API avancées Windows 32:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		AudioDevProps2.dll::2.6.4.0:A-Volute
		AudioDevProps2.dll::2.6.13.0:A-Volute
		COMCTL32.dll:Bibliothèque de contrôles de l’expérience utilisateur:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Crypto API32:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		CoreMessaging.dll:Microsoft CoreMessaging Dll:10.0.22000.71 (WinBuild.160101.0800):Microsoft Corporation
		CorperfmonExt.dll:Microsoft Common Language Runtime - Performance Counter DLL:4.8.4161.0 built by: NET48REL1:Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		DEVOBJ.dll:Device Information Set DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DNS DLL de l’API Client:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		GLU32.dll:DLL bibliothèque des utilitaires OpenGL:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		HID.DLL:Bibliothèque d’utilisateur IHM:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:API de l’application d’assistance IP:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:DLL du client API BASE Windows NT:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:DLL du client API BASE Windows NT:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		MSASN1.dll:ASN.1 Runtime APIs:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		MSCTF.dll:DLL de MSCTF Server:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.22000.184 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE pour Windows:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		OleAut32.dll:OLEAUT32.DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:DLL d’application d’assistance Windows pour les données de performance:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Runtime d’appel de procédure distante:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		SETUPAPI.DLL:Installation de L’API Windows:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:DLL commune du shell Windows:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll:User Mode Power Dependency Coordinator:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:DLL client de l’API uilisateur de Windows multi-utilisateurs:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.29.30133.0 built by: vcwrkspc:Microsoft Corporation
		VCRUNTIME140_CLR0400.dll:Microsoft® C Runtime Library:14.10.25028.0 built by: VCTOOLSD15RTM:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Services HTTP Windows:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:DLL API MCI:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		WINSTA.dll:Winstation Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		WINTRUST.dll:Microsoft Trust Verification APIs:10.0.22000.527 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:Windows Socket 2.0 32-Bit DLL:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		WSOCK32.dll:Windows Socket 32-Bit DLL:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		WTSAPI32.dll:Windows Remote Desktop Session Host Server SDK APIs:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.22000.120 (WinBuild.160101.0800):Microsoft Corporation
		bcrypt.dll:Bibliothèque de primitives de chiffrement Windows:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.22000.376 (WinBuild.160101.0800):Microsoft Corporation
		cfgmgr32.dll:Configuration Manager DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM pour Windows:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		cryptnet.dll:Crypto Network Related API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		d3d11.dll:Direct3D 11 Runtime:10.0.22000.527 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc.DLL:Service client DHCP:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc6.DLL:Client DHCPv6:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		dinput8.dll:Microsoft DirectInput:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		directxdatabasehelper.dll:DirectXDatabaseHelper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		drvstore.dll:Driver Store API:10.0.22000.120 (WinBuild.160101.0800):Microsoft Corporation
		dwmapi.dll:API du Gestionnaire de fenêtres du Bureau Microsoft:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		dxcore.dll:DXCore:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		dxgi.dll:DirectX Graphics Infrastructure:10.0.22000.469 (WinBuild.160101.0800):Microsoft Corporation
		eamsi.dll:ESET Antimalware Scan Interface DLL:10.24.19.0:ESET
		fwpuclnt.dll:API en mode utilisateur FWP/IPsec:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.22000.527 (WinBuild.160101.0800):Microsoft Corporation
		glfw.dll
		icm32.dll:Microsoft Color Management Module (CMM):10.0.22000.469 (WinBuild.160101.0800):Microsoft Corporation
		imagehlp.dll:Windows NT Image Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		inputhost.dll:InputHost:10.0.22000.168 (WinBuild.160101.0800):Microsoft Corporation
		java.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		javaw.exe:OpenJDK Platform binary:17.0.1.0:Microsoft
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		jli.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		jna9057980902474583930.dll:JNA native library:6.1.1:Java(TM) Native Access (JNA)
		jvm.dll:OpenJDK 64-Bit server VM:17.0.1.0:Microsoft
		kernel.appcore.dll:AppModel API Host:10.0.22000.71 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		lwjgl_opengl.dll
		lwjgl_stb.dll
		management.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		management_ext.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		mscms.dll:DLL du système de correspondance de couleurs Microsoft:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		mscoree.dll:Microsoft .NET Runtime Execution Engine:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		mscoreei.dll:Microsoft .NET Runtime Execution Engine:4.8.4341.0 built by: NET48REL1LAST_C:Microsoft Corporation
		msscntrs.dll:PKM Perfmon Counter DLL:7.0.22000.527 (WinBuild.160101.0800):Microsoft Corporation
		msvcp140.dll:Microsoft® C Runtime Library:14.29.30133.0 built by: vcwrkspc:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Fournisseur de service Sockets 2.0 de Microsoft Windows:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:Fournisseur Shim d’affectation de noms de messagerie:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Routeur Windows NCrypt:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		nio.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		nlansp_c.dll:NLA Namespace Service Provider DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		ntdll.dll:DLL Couche NT:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		ntmarta.dll:Fournisseur MARTA Windows NT:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		nvapi64.dll:NVIDIA NVAPI Library, Version 511.79 :30.0.15.1179:NVIDIA Corporation
		nvldumdx.dll:NVIDIA Driver Loader, Version 511.79 :30.0.15.1179:NVIDIA Corporation
		nvoglv64.dll:NVIDIA Compatible OpenGL ICD:30.0.15.1179:NVIDIA Corporation
		nvspcap64.dll:NVIDIA Game Proxy:3.25.0.84:NVIDIA Corporation
		nvwgf2umx.dll:NVIDIA D3D10 Driver, Version 511.79 :30.0.15.1179:NVIDIA Corporation
		opengl32.dll:OpenGL Client DLL:10.0.22000.41 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:DLL d’objets Performances système Windows:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		perfproc.dll:DLL d’objets Performances de processus système Windows:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		pfclient.dll:SysMain Client:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:Fournisseur d’espace de noms PNRP:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		powrprof.dll:DLL d’assistance du profil d’alimentation:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.22000.184 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.22000.184 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Bibliothèque d’utilitaires légers du Shell:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		svml.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		textinputframework.dll:"TextInputFramework.DYNLINK":10.0.22000.282 (WinBuild.160101.0800):Microsoft Corporation
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		ucrtbase_clr0400.dll:Microsoft® C Runtime Library:14.10.25028.0 built by: VCTOOLSD15RTM:Microsoft Corporation
		uxtheme.dll:Bibliothèque de thèmes Ux Microsoft:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.29.30133.0 built by: vcwrkspc:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
		vmPerfmon.dll:vmwarePerfmon DLL:16.2.2 build-19200509:VMware, Inc.
		win32u.dll:Win32u:10.0.22000.527 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:API de stockage Microsoft WinRT:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		wintypes.dll:DLL de types de base Windows:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		wldp.dll:Stratégie de verrouillage Windows:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.22000.41 (WinBuild.160101.0800):Microsoft Corporation
		xinput1_4.dll:API du contrôleur commun Microsoft:10.0.22000.493 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:17.0.1.0:Microsoft
Stacktrace:
	at net.minecraft.client.main.Main.main(Main.java:199)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:416)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:77)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.18.1
	Minecraft Version ID: 1.18.1
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.1, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1460351576 bytes (1392 MiB) / 2617245696 bytes (2496 MiB) up to 6308233216 bytes (6016 MiB)
	CPUs: 16
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i9-10980HK CPU @ 2.40GHz
	Identifier: Intel64 Family 6 Model 165 Stepping 2
	Microarchitecture: unknown
	Frequency (GHz): 3.10
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: Intel(R) UHD Graphics
	Graphics card #0 vendor: Intel Corporation (0x8086)
	Graphics card #0 VRAM (MB): 1024.00
	Graphics card #0 deviceId: 0x9bc4
	Graphics card #0 versionInfo: DriverVersion=26.20.100.7985
	Graphics card #1 name: NVIDIA GeForce RTX 2080 Super
	Graphics card #1 vendor: NVIDIA (0x10de)
	Graphics card #1 VRAM (MB): 4095.00
	Graphics card #1 deviceId: 0x1e93
	Graphics card #1 versionInfo: DriverVersion=30.0.15.1179
	Memory slot #0 capacity (MB): 16384.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 16384.00
	Memory slot #1 clockSpeed (GHz): 3.20
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 38184.21
	Virtual memory used (MB): 19912.67
	Swap memory total (MB): 5632.00
	Swap memory used (MB): 111.66
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx6016m -Xms256m
	Fabric Mods: 
		advancements-enlarger: Advancements Enlarger 0.2.4
		amecsapi: Amecs API 1.3.4+mc1.18-pre1
		appleskin: AppleSkin 2.3.0+mc1.18.1
		architectury: Architectury 3.7.20
		balm-fabric: Balm 2.4.1+0
		beehivetooltips: Beehive Tooltips 1.4.1
		betterbiomeblend: Better Biome Blend 1.18.1-1.3.2-fabric
		betterfpsdist: Better FPS distance Mod 1.18-1.6
		brb: Better Recipe Book 1.5.6
		clickadv: Clickable Advancements Mod 2.3
		clienttweaks: Client Tweaks 7.0.0
		cloth-basic-math: cloth-basic-math 0.6.0
		cloth-config: Cloth Config v6 6.2.57
		clumps: Clumps 8.0.0+5
		coat: Coat 1.0.0-beta.15
		com_electronwill_night-config_core: core 3.6.3
		com_electronwill_night-config_toml: toml 3.6.3
		controlling: Controlling For Fabric 9.0+15
		durability101: Durability101 0.0.4
		dynview: Dynamic View Mod 1.18-1.5
		entityculling: EntityCulling-Fabric 1.5.0
		fabric: Fabric API 0.46.4+1.18
		fabric-api-base: Fabric API Base 0.4.2+d7c144a865
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.5.3+d7c144a865
		fabric-biome-api-v1: Fabric Biome API (v1) 6.0.2+d7c144a865
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.10+3ac43d9565
		fabric-command-api-v1: Fabric Command API (v1) 1.1.7+d7c144a865
		fabric-commands-v0: Fabric Commands (v0) 0.2.6+b4f4f6cd65
		fabric-containers-v0: Fabric Containers (v0) 0.1.19+d7c144a865
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.4.8+d7c144a865
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.9+3ac43d9565
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.10+a1d9bbf565
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.6+d7c144a865
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.17+d7c144a865
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.9+d7c144a865
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.11+d7c144a865
		fabric-item-api-v1: Fabric Item API (v1) 1.3.1+691a79b565
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.6+3ac43d9565
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.9+d7c144a865
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.7+b4f4f6cd65
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.13+713c266865
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.9+d7c144a865
		fabric-mining-level-api-v1: Fabric Mining Level API (v1) 1.0.6+d7c144a865
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.11+b4f4f6cd65
		fabric-models-v0: Fabric Models (v0) 0.3.4+d7c144a865
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.19+d7c144a865
		fabric-networking-v0: Fabric Networking (v0) 0.3.6+b4f4f6cd65
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.11.4+d7c144a865
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.12+d7c144a865
		fabric-particles-v1: Fabric Particles (v1) 0.2.10+526dc1ac65
		fabric-permissions-api-v0: fabric-permissions-api 0.1-SNAPSHOT
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.2+ad01bfbd65
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.11+b0b66fc365
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.15+6825030165
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.10+b4f4f6cd65
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.5+d7c144a865
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.19+3ac43d9565
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.12+b4f4f6cd65
		fabric-rendering-v1: Fabric Rendering (v1) 1.10.6+713c266865
		fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 1.0.2+d7c144a865
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.14+713c266865
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.8+d7c144a865
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.12+d7c144a865
		fabric-structure-api-v1: Fabric Structure API (v1) 2.1.3+d7c144a865
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.2.8+d7c144a865
		fabric-textures-v0: Fabric Textures (v0) 1.0.10+3ac43d9565
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.3.8+fb3b57b465
		fabric-transfer-api-v1: Fabric Transfer API (v1) 1.5.9+d7c144a865
		fabricloader: Fabric Loader 0.13.0
		ferritecore: FerriteCore 4.1.2
		forgeconfigapiport: Forge Config API Port 3.1.1
		goosik-config: goosik-config 0.2.0
		iceberg: Iceberg 1.0.38
		illuminations: Illuminations 1.10.2
		infinitemusic: Infinite Music 1.5.1
		inventoryhud: Inventory HUD + 3.4.2
		java: OpenJDK 64-Bit Server VM 17
		legendarytooltips: Legendary Tooltips 1.2.4
		litematica: Litematica 0.10.2
		malilib: MaLiLib 0.11.7
		minecraft: Minecraft 1.18.1
		minihud: MiniHUD 0.21.3
		modmenu: Mod Menu 3.0.1
		mousewheelie: Mouse Wheelie 1.8.2+mc1.18-pre5
		notenoughcrashes: Not Enough Crashes 4.1.4+1.18.1
		replaymod: Replay Mod 1.18.1-2.6.3
		sound_physics_remastered: Sound Physics Remastered 1.18.1-1.0.4
		spark: spark 1.8.6
		tweed4_annotated: tweed4_annotated 1.2.0
		tweed4_base: tweed4_base 1.3.1
		tweed4_data: tweed4_data 1.2.0
		tweed4_data_hjson: tweed4_data_hjson 1.1.0
		tweed4_tailor_coat: tweed4_tailor_coat 1.0.2
		tweed4_tailor_screen: tweed4_tailor_screen 1.0.2
		visuality: Visuality 0.4.1
		voicechat: Simple Voice Chat 1.18.1-2.2.16
		worldedit: WorldEdit 7.2.9+e20e013
		xaerominimap: Xaero's Minimap 22.2.0.1
		xaeroworldmap: Xaero's World Map 1.20.3.1
	Launched Version: fabric-loader-0.13.0-1.18.1
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: Unknown
	Window size: <not initialized>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: <disabled>
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: <unknown>
	Suspected Mods: Minecraft (minecraft), Fabric Loader (fabricloader)
```

**Comment by Siphalor** [comment]
Thanks for the report.

This crash is a crash between Fabric API, ReplayMod and Mouse Wheelie and is already being tracked as #155.

**Comment by Siphalor** [comment]
Just a note that this is fixed in the latest release (1.8.3)
